package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "design-system",
+  "version": "1.0.0",
+  "description": "Design system of MB",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chhavijustme/design-system.git"
+  },
+  "keywords": [
+    "design-system"
+  ],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/chhavijustme/design-system/issues"
+  },
+  "homepage": "https://github.com/chhavijustme/design-system#readme",
+  "dependencies": {
```

Comment:
This section needs some thought. When we integrate this with MB, MB's package.json will probably depend on `design-system`, and then have imports like `import Button from 'design-system/components/Button';`. But MB will be using its own version of React. It would be very bad if that version differed from the one required here, because we'd end up bundling or executing multiple versions of React side-by-side, which can be a source of bugs in addition to adding a lot of bloat.

Here's a helpful blog post if you want more information about this problem: https://yarnpkg.com/blog/2018/04/18/dependencies-done-right/

So here's what I think might work:
 * Move everything in `dependencies` except react, react-bootstrap, and react-dom to `devDependencies`, since they're all required while developing but not executing the library.
* Add react, react-bootstrap, and react-dom to `peerDependencies` since they're needed to execute the components but should use the exact same instances required by MB.
* Remove the `dependencies` section since I don't think we have any dependencies that aren't peer dependencies.